tests/bootstrap.php

```diff
@@ -9,3 +9,5 @@
 } else {
     throw new Exception('Can\'t find autoload.php. Did you install dependencies with Composer?');
 }
+
+class_alias("PHPUnit\Framework\Error\Warning", "PHPUnit_Framework_Error_Warning");
```

Comment:
This is going to fail on PHPUnit 4 (PHP 5.x), as the original class (first argument) doesn't exist. You'll also get a warning because [`class_alias()`](http://php.net/manual/en/function.class-alias.php) will try to autoload that class and fail to find it (controlled by its third boolean argument).

Instead, I'd suggest using the PHPUnit 6 class names in our tests and only creating the alias when needed for compatibility with PHPUnit 4:

```php
if ( ! class_exists('PHPUnit\Framework\Error\Warning')) {
    class_alias('PHPUnit_Framework_Error_Warning', 'PHPUnit\Framework\Error\Warning');
}
```

This condition should only be hit by PHPUnit 4, when the new class isn't found (even after an autoloading attempt). It will then create the `PHPUnit\Framework\Error\Warning` alias for the old `PHPUnit_Framework_Error_Warning` class, which will also be autoloaded by default.

AFAIK, the only references to "PHPUnit_Framework_Error_Warning" are in:

 * tests/GridFS/BucketFunctionalTest.php
 * tests/GridFS/SpecFunctionalTest.php

Most are in `@expectedException` annotations but at least one is a string.